test/data/html/tests.js

```diff
@@ -473,8 +473,8 @@ exports.test_data = {
             output: '<?xml version="1.0" encoding="UTF-8" ?>\n<root{{indent_attr_first}}attr1="foo"{{indent_attr}}{{indent_attr_faligned}}attr2="bar"{{indent_end_selfclosing}}/>'
         }, {
             fragment: true,
-            input: '<link href="//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,400,600,700,300&amp;subset=latin" rel="stylesheet" type="text/css">',
```

Comment:
Rather than replacing the existing test with a different one, keep the existing test(s) and then add the new test(s).